### DISCUSSION TITLE
求助23.05分支IPv6问题

### DISCUSSION THREAD

**Discussion by nantayo** [immortalwrt/immortalwrt#1376] [Q&A] [answered]
&ensp;&ensp;&ensp;&ensp;我在使用23.05分支时遇到了IPv6能够下发，但却无法访问互联网的问题，具体情况介绍如下：

&ensp;&ensp;&ensp;&ensp;我使用一台X86-64设备，安装了自行编译的23.05 snapshot分支，版本号为r0-d201607，含有ipv6helper等必备软件包，额外插件有：mosdns、passwall、docker、samba、zerotier。当PPPoE拨号时，设备本身能够正常获取到宽带的IPv6-PD信息，局域网各设备也可以获取到IPv6地址等信息，但无法访问IPv6域名或者ping通IPv6地址。

&ensp;&ensp;&ensp;&ensp;首先我的DHCP有关IPv6部分的设置方式是：RA服务混合模式；通告本地IPv6 DNS服务器；DHCPv6服务、NDP代理禁用；默认路由器自动；启用SLAAC；RA标记无。

&ensp;&ensp;&ensp;&ensp;其次，为了研究问题详情我做了如下测试：
1. 使用一台没有刷机的硬路由在全默认状态下进行PPPoE拨号，发现局域网设备IPv6使用一切正常，说明宽带具备完善的IPv6功能；
2. 将固件初始化，在不开启任何插件的情况下进行PPPoE拨号，发现不能正常使用IPv6功能；
3. SSH连接到路由器，使用ping -6测试了"www.baidu.com"以及"ipw.cn"两个域名，发现能够ping通；
4. 电脑以及手机浏览器分别访问"ipw.cn"，发现均显示IPv6未开启；
5. 电脑使用ping -6测试了"www.baidu.com"以及"ipw.cn"两个域名，发现不能ping通；
6. 查询IPv6路由信息，结果如图：
![1](https://github.com/user-attachments/assets/6afd16fd-b874-4406-a3b7-47556aad64c3)

如有其它需要的信息请告知我补充，先行谢过。

**Comment by 1715173329** [comment, marked as answer]
https://github.com/openwrt/packages/commit/55e689176ec52afde0cda0ee2b48dbff349d6c6c

**Reply by nantayo** [reply]
> [openwrt/packages@55e6891](https://github.com/openwrt/packages/commit/55e689176ec52afde0cda0ee2b48dbff349d6c6c)

看到本仓库1小时前已同步更新，那我编译试下。

**Comment by nantayo** [comment]
确认确为Docker导致的问题，更新后恢复正常
